README.md

```diff
@@ -8,10 +8,17 @@ GeolocatorService aims to simplify getting the user's current location and handl
 
 ## Getting Started
 
-Add the GeolocatorService nuget package to your project heads.
+#### Xamarin / UWP
+Add the GeolocatorService and the GeolocatorService.Abstractions nuget package to your project heads.
```

Comment:
You don't need to add both packages in the project heads.
The Abstractions package can be used in UI-agnostics libraries (such as netstandard2.0 or net6.0) that need the `IGeolocatorService` abstraction.

Projects that reference the `GeolocatorService` package don't need to add a reference to the abstractions package because they already have it transitively.